app/models/finance/billing_strategy.rb

```diff
@@ -349,18 +345,12 @@ def needs_credit_card?
 
   private
 
-  # Yields a block for each buyer, passing it as a parameter. If an
-  # exception occurs meanwhile, catches it and reports by airbrake.
-  #
+  # Yields a block for each buyer, passing it as a parameter.
   def bill_and_charge_each(options = {})
     buyer_ids = options[:buyer_ids]
     @failed_buyers = []
 
-    if provider.nil?
-      airbrake(:error_message => "WARNING: tried to use billing strategy #{self.id} which has no account",
-               :error_class => 'InvalidData')
-      return
-    end
+    return if provider.nil?
```

Comment:
```suggestion
    return unless provider
```

Imo it should become the first line of the method now :smile: